### PR TITLE
sig-release: Add Lauri Apple as SIG Chair

### PR DESCRIPTION
**Pull request by justaugustus**
@LappleApple has been crushing it as SIG Release's (and the community's first _named_) Program Manager for some time now.

~...but our generator scripts do not currently support SIG titles outside of `Chair`, `Technical Lead`, and `Emeritus Lead`.~

~This PR does two things:~
~- Add support for SIG Program Managers in generator scripts~
~- Formally recognizes Lauri Apple as SIG Release's Program Manager~

EDIT (2021-04-21):

...however, due to the [community limitations around formally recognizing this work](https://github.com/kubernetes/community/issues/5722), @kubernetes/sig-release-leads would like to nominate Lauri as a Chair for SIG Release.

/assign @saschagrunert @LappleApple @nikhita 
cc: @kubernetes/sig-release-leads @kubernetes/sig-contributor-experience-leads @kubernetes/steering-committee

### CODE REVIEW

File: generator/sig_readme.tmpl
Comment (justaugustus) [inline generator/sig_readme.tmpl:46]: @LappleApple -- this is something I word-souped. Do you want to put your own flair on this description?

**Comment by LappleApple** [comment]
Thank you for the kind words, Stephen. Seems like a helpful step toward supporting the program manager role in other SIGs—a proposal that could lead to several benefits for overloaded Chairs and TLs.

**Comment by dims** [comment]
I'd definitely support this. as a pre-req can you please add a short description here:
https://github.com/kubernetes/community/tree/master/contributors/chairs-and-techleads

and update here:
https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md

thanks,
Dims

/hold

**Comment by justaugustus** [comment]
> As a pre-req can you please add a short description here:
> https://github.com/kubernetes/community/tree/master/contributors/chairs-and-techleads
> 
> and update here:
> https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md

@LappleApple -- Would you mind working on this?

**Review by nikhita — COMMENTED** [review]
LGTM for the generator changes.

I'm also good with documenting the program manager role in the governance docs as a follow-up. Lauri has already been doing a lot for SIG Release and it is evident that having a program manager role has value.

I don't think we need to block this PR until we get the docs added (just need to get it done soon after this PR merges) but will leave it to the rest of the steering committee to ack.

**Comment by nikhita** [comment]
@LappleApple saw your 👍 in https://github.com/kubernetes/community/pull/5563#issuecomment-785846652. Let's go by the book and get the roles in first then :)

**Comment by derekwaynecarr** [comment]
+1 to definition of the role in governance docs first.  i support the recognizing the value the role can provide to SIGs.

**Comment by justaugustus** [comment]
> As a pre-req can you please add a short description here:
> https://github.com/kubernetes/community/tree/master/contributors/chairs-and-techleads
> 
> and update here:
> https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md

> +1 to definition of the role in governance docs first. i support the recognizing the value the role can provide to SIGs.

@dims @derekwaynecarr -- @LappleApple just opened https://github.com/kubernetes/community/pull/5565, with loads of goodies, including an extensive PgM description.

**Comment by LappleApple** [comment]
PTAL: [#5565](https://github.com/kubernetes/community/pull/5565)

File: generator/sig_readme.tmpl
Comment (LappleApple) [reply generator/sig_readme.tmpl:46]: The flair has been put!

**Comment by parispittman** [comment]
love this; what's the main differentiator between this role and chairs?

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by justaugustus** [comment]
@kubernetes/steering-committee -- Where did we land with this discussion?

**Comment by dims** [comment]
@justaugustus https://github.com/kubernetes/community/issues/5722

**Comment by justaugustus** [comment]
Conferring with the @kubernetes/sig-release-leads, I've updated this PR to propose @LappleApple as a SIG Release Chair.
I will send a note to SIG Release shortly.

We're establishing a lazy consensus timeout for this decision of Thursday, April 29 4pm US Eastern.

cc: @kubernetes/sig-release

**Review by saschagrunert — APPROVED** [review]
I think there is nothing more to add.

/lgtm

**Review by hasheddan — APPROVED** [review]
/lgtm

Thank you for everything you have done and continue to do @LappleApple!

**Comment by cpanato** [comment]
thanks and congratulations @LappleApple

**Review by xmudrii — APPROVED** [review]
/lgtm

Congrats Lauri! Thank you for all the amazing work you're doing for SIG Release!

**Comment by puerco** [comment]
This is awesome, @LappleApple has completely redone the way we do things in SIG Release, making us much more productive and able to achieve our goals. Thank you Lauri, you're the best :)
/lgtm

**Comment by reylejano** [comment]
Lauri has done amazing work to improve SIG Release 
/lgtm

**Comment by cblecker** [comment]
@saschagrunert @justaugustus: Can you please ensure that the [process around leadership changes](https://github.com/kubernetes/community/blob/master/contributors/chairs-and-techleads/leadership-changes.md) is followed? I haven't seen anything out to k-dev yet. Thank you 🙏 !

/hold
explicit hold for both governance process and consensus period

**Comment by LappleApple** [comment]
I appreciate this offer, but for personal reasons have decided to decline this opportunity to continue doing program management work in Release and elsewhere across the project.

**Comment by justaugustus** [comment]
No worries, @LappleApple!
Thanks for taking the time to consider this and thank you for the continued value you bring to the community, both as a person and as our Program Manager! 

/close

**Comment by jayunit100** [comment]
oh too bad ?

**Pull request closed without merging**
